src/test/java/seedu/address/logic/commands/ListUnvisitedCommandTest.java

```diff
@@ -30,12 +27,12 @@ public void setUp() {
         expectedModel.updateFilteredRestaurantList(Model.PREDICATE_SHOW_UNVISITED_RESTAURANTS);
 
     }
-
-    @Test
```

Comment:
Perhaps you could fix this test for the next milestone as this test is currently empty